### PR TITLE
Attempt at fixing disappearing icons

### PR DESCRIPTION
**Pull request by jivinivan**

### CODE REVIEW

**Comment by jivinivan** [comment]
.

**Comment by jivinivan** [comment]
@Chochek please review

**Comment by jivinivan** [comment]
[(0) Reactions: Attempt at fixing disappearing icons in Chrome [1]](https://trello.com/c/TPfVbQVq/2755-0-reactions-attempt-at-fixing-disappearing-icons-in-chrome-1)

File: src/components/Icon/Icon.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ const Icon = props => {
       fill={fill}
       className={className}
       onClick={onClick}
-      viewBox='0 0 24 24'>
+      viewBox='0 0 24 24'
+      xmlns="http://www.w3.org/2000/svg">
```
Comment (fieldsco) [inline src/components/Icon/Icon.js]:
```suggestion
      xmlns='http://www.w3.org/2000/svg'>
```

**Comment by jivinivan** [comment]
Didn't work

**Pull request closed without merging**